jinahub/encoders/audio/AudioCLIPEncoder/README.md

```diff
@@ -1,8 +1,7 @@
 # AudioCLIPEncoder
 
-**AudioCLIPEncoder** is a class that wraps the [AudioCLIP](https://github.com/AndreyGuzhov/AudioCLIP) model for generating embeddings for audio data. 
+**AudioCLIPEncoder** is a class that wraps the [AudioCLIP](https://github.com/AndreyGuzhov/AudioCLIP) model for generating embeddings for audio data.
 
+This encoder is meant to be used in conjunction with the [AudioCLIPImageEncoder](https://github.com/jina-ai/executors/tree/main/jinahub/encoders/image/AudioCLIPImageEncoder) and [AudioCLIPTextEncoder](https://github.com/jina-ai/executors/tree/main/jinahub/encoders/text/AudioCLIPTextEncoder), as it can embedd text, images and audio to the same latent space.
```

Comment:
```suggestion
This encoder is meant to be used in conjunction with the [AudioCLIPTextEncoder](https://hub.jina.ai/executor/jfe8kovq) and [AudioCLIPTextEncoder](https://hub.jina.ai/executor/jfe8kovq), as it can embed text, images and audio to the same latent space.
```